pm_muairss_write/get_out_folder.py

```diff
@@ -0,0 +1,11 @@
+import sys
+
+import yaml
+
+
+try:
+    with open('params.yaml') as f:
+        data = yaml.safe_load(f)
+    sys.stdout.write(data['out_folder'])
+except KeyError or FileNotFoundError:
```

Comment:
Oh, we should remove the FileNotFoundError from here too, I don't think it got copied from the other PRs. (Hopefully it shouldn't ever be hit since the input comes direct from the Galaxy interface in this case)